### ISSUE TITLE
Warning: Unknown props `formulas`, `component`

### ISSUE THREAD

**Issue by toannguyen132** [gilbox/react-track#21]
I got a warning on my browser (Chrome) when I use react-track. Do you know why this happens?
The full warning message is:
`Warning: Unknown props`formulas`,`component`on <div> tag. Remove these props from the element.`
Here is my code:

```
<TrackDocument formulas={[topTop,topBottom,calculateScrollY]}>
  {(topTop, topBottom, scrollY) =>
    <TrackedDiv formulas={[topBottom]}>
      {(posBottom) =>
        <div className={cx("services-list", {
          'active':scrollY > (posBottom),
  })}>
```

Thanks in advance.

**Comment by daaain** [comment]
This is a duplicate of #19.

**Comment by daaain** [comment]
Fixed in v0.4.1 now, please update!

**Comment by daaain** [comment]
Actually make it v0.4.2, I managed to publish before doing a build (and NPM doesn't allow republishing) :facepalm:

**Comment by gilbox** [comment]
@daaain fyi -- I suspect you ran `npm publish` instead of `npm run publish`. There is a big difference between the two. I just published `0.4.3`. Hopefully it's all good now.

**Comment by daaain** [comment]
@gilbox I think that might have been the case, should have done it a bit more carefully, sorry :(
